packages/trpc/server/routers/viewer/me/getUserTimePreferences.handler.ts

```diff
@@ -0,0 +1,19 @@
+import type { Session } from "next-auth";
+
+import type { TrpcSessionUser } from "@calcom/trpc/server/types";
+
+type GetUserTimePreferencesOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+    session: Session;
+  };
+};
+
+export const getUserTimePreferencesHandler = async ({ ctx }: GetUserTimePreferencesOptions) => {
+  const { user } = ctx;
```

Comment:
I created this new handler to get timeFormat and timeZone information. I don't have this information from `useSession()`, but didn't want to use `useMeQuery()` which does too much. I didn't find any other existing variables or hooks that provides this information. Let me know if it exists!